Explorer/Assets/DCL/AvatarRendering/Wearables/Systems/ResolveWearableByPointerSystem.cs

```diff
@@ -114,11 +108,46 @@ private void ResolveWearablePromise([Data] bool defaultWearablesResolved, in Ent
                 return;
             }
 
+            if (successfulDtos == wearablesByPointersIntention.Pointers.Count)
+            {
+                ref HideWearablesResolution hideWearablesResolution = ref wearablesByPointersIntention.HideWearablesResolution;
+
+                if(hideWearablesResolution.VisibleWearables == null)
+                    CalculateVisibleWearables(resolvedWereables.ToArray(), wearablesByPointersIntention.BodyShape, ref hideWearablesResolution);
```

Comment:
Avoid `ToArray`: just pass `resolvedWereables` as `IReadOnlyList`: you don't intend to modify it anyway